sklearn/inspect/partial_dependence.py

```diff
@@ -179,6 +204,14 @@ def partial_dependence(est, features, X, percentiles=(0.05, 0.95),
         ``X`` is used both to generate a grid of values for the
         ``features``, and to compute the averaged predictions when
         method is 'brute'.
+    response : 'auto', 'proba' or 'decision', optional (default='auto') :
```

Comment:
I'd be more inclined to have values `'auto'`, `'predict_proba'` and `'decision_function'` as its translation is more explicit.

I'm not persuaded by the name `response` either. `response_method` is clearer...